domain/consensus/processes/dagtraversalmanager/block_heap.go

```diff
@@ -75,7 +75,7 @@ func (dtm dagTraversalManager) NewDownHeap() model.BlockHeap {
 		dbContext:     dtm.databaseContext,
```

Comment:
dtm.NewDownHeap should also get receiver by pointer (and maybe other dtm methods as well?)